### DISCUSSION TITLE
HOw to convert the dbresnet50 file tensorflow to onnx file

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1224] [General]
I wanted to know how to convert these files to onnx .Can anyone help please! @felixT2K ..PLease share the code to convert it 
![image](https://github.com/mindee/doctr/assets/122434059/3f9b7f48-ee66-4199-9c35-1d2ffc8d5519)

**Comment by felixT2K** [comment]
Hey :) take a look at: https://github.com/mindee/doctr/blob/9bf970e9e8f3823d7343890bbc9f321f93261b93/tests/tensorflow/test_models_detection_tf.py#LL194C66-L194C66
